dev-docs/bidders/aniview.md

```diff
@@ -21,29 +25,144 @@ For more information about [Aniview Ad Server](https://www.aniview.com/), please
-|------------------|----------|------------------|------------------------------|----------|
-| `AV_PUBLISHERID` | required | Publisher/Netid  | `'55b88d4a181f465b3e8b4567'` | `string` |
-| `AV_CHANNELID`   | required | Channel id       | `'5a5f17a728a06102d14c2718'` | `string` |
+| Name             | Scope    | Description           | Example              | Type     |
+|------------------|----------|-----------------------|----------------------|----------|
+| `AV_PUBLISHERID` | required | Publisher/Network ID  | `'Get from Aniview'` | `string` |
+| `AV_CHANNELID`   | required | Channel ID            | `'Get from Aniview'` | `string` |
 
-### Test Parameters
+### Setup for Video
 
 ```javascript
-const videoAdUnit = [
-{
-  code: 'video1',
+const adUnit = [{
+  code: 'videoAdUnit',
   mediaTypes: {
     video: {
+      // Required
       playerSize: [[640, 480]],
-      context: 'outstream'
+      context: 'outstream',
+      mimes: ['video/mp4', 'video/mpeg', 'application/javascript'],
+      
+      // Optional
+      playbackmethod: [1, 2],
+      protocols: [1, 2, 3, 5, 6, 7, 8],
+      api: [1, 2],
+      maxduration: 60,
+      plcmt: 4,
     },
   },
   bids: [{
     bidder: 'aniview',
     params: {
-      AV_PUBLISHERID: '55b78633181f4603178b4568',
-      AV_CHANNELID: '5d19dfca4b6236688c0a2fc4'
-    }
-  }]
+      // Required
+      AV_PUBLISHERID: 'Get from Aniview',
+      AV_CHANNELID: 'Get from Aniview',
+    },
+  }],
+}];
+```
+
+### Setup for Banner
+
+```javascript
+const adUnit = [{
+  code: 'bannerAdUnit',
+  mediaTypes: {
+    banner: {
+      // Required
+      sizes: [[300, 250], [300, 600]],
+    },
+  },
+  bids: [{
+    bidder: 'aniview',
+    params: {
+      // Required
+      AV_PUBLISHERID: 'Get from Aniview',
+      AV_CHANNELID: 'Get from Aniview',
+    },
+  }],
 }];
 ```
+
+### Setup for Multi-format (Banner & Video)
+
+```javascript
+const adUnit = [{
+  code: 'multiformatAdUnit',
+  mediaTypes: {
+    banner: {
+      // Required
+      sizes: [[300, 250], [300, 600]],
+    },
+    video: {
+      // Required
+      playerSize: [[640, 480]],
+      context: 'outstream', 
+      mimes: ['video/mp4', 'video/mpeg', 'application/javascript'],
+    },
+  },
+  bids: [{
+    bidder: 'aniview',
+    params: {
+      // Required
+      AV_PUBLISHERID: 'Get from Aniview',
+      AV_CHANNELID: 'Get from Aniview',
+    },
+  }],
+}];
+```
+
+### Floors
+
+```javascript
+const adUnit = [{
+  code: 'floorsAdUnit',
+  mediaTypes: {/*...*/},
+  bids: [/*...*/],
+  floors: {
+    currency: 'USD',
+    schema: {
+      delimiter: '|',
+      fields: ['mediaType', 'size'],
+    }, 
+    values: {
+      'banner|300x250': 0.1,
+      'banner|*': 0.05,
+      'video|640x360': 0.5,
+      'video|*': 0.25,
+    },
+  },
+}];
+```
+
+### Cookie sync
+
+```javascript
+pbjs.setConfig({
+  // Cookie sync (Image & Iframe)
+  userSync: {
+    filterSettings: {
+      all: {
+        bidders: '*',
```

Comment:
We cannot let you tell publishers to always include all bidders. Our preference would be to drop this whole section. If necessary, you can ask publishers to sync aniview.